libekb.C

```diff
@@ -155,6 +156,49 @@ static void get_HWPErrorInfo(const fapi2::ReturnCode& rc, HWP_ErrorInfo& hwp_err
     }
 }
 
+/*
+ * @brief Used to callout hardware error details.
+ *
+ * The defined api is used collect defined error info in error xml based on
```

Comment:
Is it really doing this here , you are cutomizing right, rephrase with acual scope.

Reply:
yes, this is wrong one. (copy pasted from above, forgot to update